lib/package.rb

```diff
@@ -27,7 +27,6 @@ def require_gem(gem_name_and_require = nil, require_override = nil)
 
 def agree_with_default(yes_or_no_question_msg, character = nil, default:)
   yes_or_no_question = yes_or_no_question_msg.lightpurple
-  require_gem('highline')
```

Comment:
We use highline in the lines immediately following, so we should probably still require it in `lib/package.rb`

Reply:
Agreed, but I'm loading highline from `crew` now, so it should already be loaded. (Otherwise I was getting issues with stuff complaining about highline not being loaded.)

Reply:
Fair, although I think it's still a good idea to require the necessary files rather than relying on them being implicitly present. (i.e. if we only load lib/package.rb and it's not run through bin/crew, which is the case for all our tests.)

Reply:
Resolving since a `require_gem('highline')` is still in `lib/package.rb`.